### PR TITLE
Add work order list details

### PR DESCRIPTION
**Pull request by josh-dadswell-hackney**
## Link to JIRA ticket

https://trello.com/c/KDGxtwH2/107-as-an-agent-i-can-view-repairs-history-on-a-dwelling-so-that-i-dont-raise-duplicate-repairs

#### _Checklist_

- [x] Added end-to-end (i.e. integration) tests where necessary e.g to test complete functionality of a newly added feature
- [x] Added tests to cover all new production code
- [x] Added comments to the README or updated relevant documentation _(add link to documentation)_, where necessary.
- [x] Checked all code for possible refactoring
- [x] Code pipeline builds correctly

### CODE REVIEW

File: RepairsApi.Tests/Helpers/StubGeneration/ObjectGenerationExtensions.cs [hunk 1]
```diff
@@ -43,6 +43,7 @@ public static Generator<T> AddWorkOrderGenerators<T>(this Generator<T> generator
         {
             return generator
                 .AddDefaultGenerators()
+                .AddValue(null, (RepairsApi.V2.Infrastructure.WorkOrder wo) => wo.WorkOrderComplete)
```
Comment (kylealexanderhackney) [inline RepairsApi.Tests/Helpers/StubGeneration/ObjectGenerationExtensions.cs]: remove from this method

File: RepairsApi.Tests/Helpers/StubGeneration/ObjectGenerationExtensions.cs [hunk 2]
```diff
@@ -54,6 +55,21 @@ public static Generator<T> AddWorkOrderGenerators<T>(this Generator<T> generator
...
 
+
+        public static Generator<T> AddInfrastructureWorkOrderGenerators<T>(this Generator<T> generator)
+        {
+            return generator
+                .AddDefaultGenerators()
+                .AddValue(null, (RepairsApi.V2.Infrastructure.WorkOrder wo) => wo.WorkOrderComplete)
+                .AddValue("address,line", (RepairsApi.V2.Infrastructure.PropertyAddress addr) => addr.AddressLine)
+                .AddValue(2.0, (RepairsApi.V2.Infrastructure.Quantity q) => q.Amount)
+                .AddValue(2.0,
```
Comment (kylealexanderhackney) [inline RepairsApi.Tests/Helpers/StubGeneration/ObjectGenerationExtensions.cs]: you shouldnt need to copy these over. the generated ones were just to support mapping to infrastructure so the infrastructure itself shouldnt need to do anything special for these properties

File: RepairsApi.Tests/V2/Gateways/JobStatusUpdateGatewayTests.cs [hunk 3]
```diff
@@ -40,7 +42,6 @@ public async Task CanCreateJobStatusUpdate()
 
             // act
             await _classUnderTest.CreateJobStatusUpdate(expected);
-            await InMemoryDb.Instance.SaveChangesAsync();
```
Comment (kylealexanderhackney) [inline RepairsApi.Tests/V2/Gateways/JobStatusUpdateGatewayTests.cs:43]: why are we removing this?

File: RepairsApi/V2/Boundary/Response/WorkOrderListItem.cs [hunk 4]
```diff
@@ -2,6 +2,13 @@
 
 namespace RepairsApi.V2.Boundary.Response
 {
+    public static class WorkOrderStatus
+    {
+        public const string InProgress = "In Progress";
+        public const string Complete = "Work Compete";
+
```
Comment (kylealexanderhackney) [inline RepairsApi/V2/Boundary/Response/WorkOrderListItem.cs:9]: white space

File: RepairsApi.Tests/V2/Gateways/JobStatusUpdateGatewayTests.cs (hunk 3 above)
Comment (josh-dadswell-hackney) [reply RepairsApi.Tests/V2/Gateways/JobStatusUpdateGatewayTests.cs:43]: save changes is called in createjobstatusupdate

**Review by kylealexanderhackney — APPROVED** [review]

**Pull request merged**